sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DescribeNamespaceExec.scala

```diff
@@ -45,10 +42,12 @@ case class DescribeNamespaceExec(
     val metadata = catalog.loadNamespaceMetadata(ns)
 
     rows += toCatalystRow("Namespace Name", ns.last)
-    rows += toCatalystRow("Description", metadata.get(COMMENT_TABLE_PROP))
-    rows += toCatalystRow("Location", metadata.get(LOCATION_TABLE_PROP))
+    rows += toCatalystRow("Description", metadata.get(SupportsNamespaces.PROP_COMMENT))
+    rows += toCatalystRow("Location", metadata.get(SupportsNamespaces.PROP_LOCATION))
```

Comment:
shall we do the same in `DescribeTableExec`?

Reply:
you mean show description and location even when extended is false? correct me if i misunderstand.
It might be a little different. I think the main content of "describe table" should be schema, taking description and location as extended may be reasonable (like in hive).
WDYT?

Reply:
I mean display location and comment nicely, instead of displaying them with `Properties (...)`. We can do it in a followup.

Reply:
ok, will do that in followup.